demo/full/scripts/modules/player/catchUp.js

```diff
@@ -24,7 +24,7 @@ const CATCH_UP_CHANGE_RATE_STEP = 6;
 const CATCH_UP_SEEKING_STEP = 15;
 
 // Maximum playback rate we can set when catching up.
-const MAX_RATE = 10;
+const MAX_RATE = 5;
```

Comment:
While performing test, I thought that setting a playback rate at `10` in the worst case (even if it is never reached in reality) is a bit much.